### ISSUE TITLE
Fix current calendar export

### ISSUE THREAD

**Issue by philboeselager** [hbz/oerworldmap#1130]
The [current calendar export validation](https://icalendar.org/validator.html?url=https://oerworldmap.org/resource/?q=&filter.about.%40type=Event) shows some errors:

1. This is not an iCalendar file or URL near line # 1. Reference: RFC 5545 3.4. iCalendar Object.
2. Missing VCALENDAR object near line # 1. Reference: RFC 5545 3.4 iCalendar Object.
3. Missing start (end end) date for at least one event

**Comment by philboeselager** [comment]
When validating all currently 45 events via copy & paste from an exported file, the errors reduce to 1:

    Missing DTSTART property in VEVENT near line # 64

This is the event [OERcamp Ost](urn:uuid:7b0c8f88-379c-4103-8154-7177ceeebf43). As [the about page](https://oerworldmap.org/resource/urn:uuid:7b0c8f88-379c-4103-8154-7177ceeebf43.json) reveals, the author is @karindr :-) So could you please add a start date and an end date?

**Comment by philboeselager** [comment]
@literarymachine I think the other errors reported by the [icalendar.org validator](https://icalendar.org/validator.html) result from that the validator misses the carriage return or other Windows specific format signs. Anyway, the exporter itself is not the problem, as validating via an exported file shows. So for me, this issue can be closed. We should keep in mind, not to use icalendar.org validator via an URL. What do you think?

**Comment by literarymachine** [comment]
> What do you think?

The problem currently is that I cannot add https://oerworldmap.org/resource.ics?q=&filter.about.%40type=Event to a Google Calendar which was one of the main use cases for this export: "Der Kalender konnte unter der angegebenen URL nicht analysiert werden." It does not tell me why, though. [This](http://severinghaus.org/projects/icv/?url=https%3A%2F%2Foerworldmap.org%2Fresource.ics%3Fq%3D%26filter.about.%2540type%3DEvent) validator seems to have trouble, too.

**Comment by philboeselager** [comment]
I've checked it out: the problem is caused by the event mentioned above missing a start date. If you try to import "all events but this one", it works.

Start date, uid and date stamp have become required fields with the [re-specification of the iCalendar format](https://tools.ietf.org/html/rfc5545#section-3.6.1). So, I have implemented [this](https://github.com/hbz/oerworldmap/commit/368211c71baf437e4bf10613ff19975c77acc062) and I've added an [additional test case](https://github.com/hbz/oerworldmap/commit/ece2e3ef83b5bce26c8cb4905841358bc21f25c2).

Should work with direct import from URL after merging then.
